ArduPlane/Plane.cpp

```diff
@@ -32,3 +32,31 @@ Plane::Plane(void)
 
 Plane plane;
 AP_Vehicle& vehicle = plane;
+
+#if AP_SCRIPTING_ENABLED
+
+// returns true if vehicle is landing. Only used by Lua scripts
+bool Plane::is_landing() const
+{
+    #if HAL_QUADPLANE_ENABLED
+    
+    return plane.quadplane.in_vtol_land_descent();
+
+    #endif // HAL_QUADPLANE_ENABLED
+
+    return control_mode->is_landing();
```

Comment:
with quadplane enabled, this function has a double return which is unreachable code so please fix that. Also, watch out for whitespace on the # if, make it match with the rest of the code. This also needs to be moved to Arduplane.cpp under set_land_descent_rate() because there's already a group of functions there wrapped in AP_SCRIPTING_ENABLED

```suggestion
#if HAL_QUADPLANE_ENABLED
    if (plane.quadplane.in_vtol_land_descent()) {
        return true;
    }
#endif

    return control_mode->is_landing();
```